wcc-contentful-app/lib/wcc/contentful/app.rb

```diff
@@ -29,13 +28,6 @@ def self.configure
 
     configuration.validate!
 
-    # Inject additional configuration into WCC::Contentful
-    WCC::Contentful.configure do |config|
-      unless config.middleware.any? { |x| x.is_a? WCC::Contentful::App::Middleware::PublishAt }
```

Comment:
removed because they have this functionality built in?

Reply:
Exactly, and b/c we never actually used it